src/ucp/api/ucp_def.h

```diff
@@ -302,6 +324,32 @@ typedef struct ucp_err_handler {
 
 
 /**
+ * @ingroup UCP_EP
+ * @brief UCP address description in case of a client-server connection flow.
```

Comment:
@shamisp "in case" means that this address is used with a client-server connection establishment as opposed to the regular ucp connection establishment.